lms/services/event.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from functools import lru_cache
+from typing import TYPE_CHECKING
+
+from sqlalchemy.orm import Session
+
+from lms.models import Event, EventData, EventType, EventUser
+
+if TYPE_CHECKING:  # pragma: no cover
```

Comment:
Used for breaking the circular dependency.

Reply:
:thinking: I wonder if there's a way around this. Usually you can avoid this by having a third file when two import from each other.

So if the events live somewhere separate from the subscriber, I bet we could fix this.

So maybe:

 * `lms/events/event.py`
 * `lms/services/event.py` - Imports from above
 * `lms/events/subscribers.py` - Imports from both of the above